src/new-apps/components/NotInstalledAppListRow/NotInstalledAppListRow.test.tsx

```diff
@@ -0,0 +1,131 @@
+import { getAppsConfig } from "@saleor/config";
+import { AppListContext, AppListContextValues } from "@saleor/new-apps/context";
+import { activeApp } from "@saleor/new-apps/fixtures";
+import { InstalledApp } from "@saleor/new-apps/types";
+import Wrapper from "@test/wrapper";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import { MemoryRouter as Router } from "react-router-dom";
+
+import NotInstalledAppListRow from "./NotInstalledAppListRow";
+
+const Component = ({
+  data,
+  context,
+}: {
+  data: InstalledApp;
+  context: AppListContextValues;
+}) => (
+  <Wrapper>
+    <Router>
+      <AppListContext.Provider value={context}>
+        <NotInstalledAppListRow {...data} />
+      </AppListContext.Provider>
+    </Router>
+  </Wrapper>
+);
+
+describe("Apps NotInstalledAppListRow", () => {
+  it("displays app details when basic app data passed", () => {
+    // Arrange
+    const openAppSettings = jest.fn();
+    render(
+      <Component
+        data={{
+          app: activeApp,
+          isExternal: false,
+        }}
+        context={{
+          openAppSettings,
+        }}
+      />,
+    );
+    const name = screen.queryByText(activeApp.name as string);
+    const version = screen.queryByText(activeApp.version as string, {
+      exact: false,
+    });
+    const manifestDomain = screen.queryByText(
+      new URL(activeApp.manifestUrl as string).host,
+    );
+    const externalLabel = screen.queryByTestId("app-external-label");
+    const tunnelLabel = screen.queryByTestId("app-tunnel-label");
+
+    // Assert
+    expect(name).toBeTruthy();
+    expect(version).toBeTruthy();
+    expect(manifestDomain).toBeTruthy();
+    expect(externalLabel).toBeFalsy();
+    expect(tunnelLabel).toBeFalsy();
```

Comment:
I'd avoid `toBeTruthy`. These are UI elements so you need to check if they are visible
`toBeTruthy` is true when element is also hidden

https://github.com/testing-library/jest-dom#tobevisible